assets/config/config.yaml

```diff
@@ -1,18 +1,18 @@
 operator:
-  scyllaDBVersion: "6.2.0"
+  scyllaDBVersion: "docker.io/scylladb/scylla:2025.1.2@sha256:84e914792c61a7703ff616bf6ee6d4becbcf68845221bd2458fcab10ef64302c"
```

Comment:
Without https://github.com/scylladb/scylla-operator/issues/2455 we can't use sha. It must be set to just the version part (`2025.1.2`).